src/Protobuf/Types/Int64.elm

```diff
@@ -0,0 +1,21 @@
+module Protobuf.Types.Int64 exposing (Int32s, Int64, fromInt32s, toInt32s)
+
+import Internal.Int64
+
+
+type alias Int64 =
+    Internal.Int64.Int64
```

Comment:
doesn't this make the types opaque?

Reply:
Yes I like that. So it is good if people know it's internal structure (in case they want to map it to an Int64 lib for example).
Can you move the module to be `Protobuf.Types`?

Reply:
Sure, but then I'd need to rename the functions to `int64FromInt32s` and `int64ToInt32s` right?
Or do you think there is no need for that?

Reply:
Ah good point, better leave it as is then.